packages/kbn-apm-synthtrace/src/lib/logs/logs_synthtrace_es_client.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { Client } from '@elastic/elasticsearch';
+import { dedot, ESDocumentWithOperation } from '@kbn/apm-synthtrace-client';
+import { pipeline, Readable, Transform } from 'stream';
+import { LogDocument } from '@kbn/apm-synthtrace-client/src/lib/logs';
+import { SynthtraceEsClient, SynthtraceEsClientOptions } from '../shared/base_client';
+import { getSerializeTransform } from '../shared/get_serialize_transform';
+import { Logger } from '../utils/create_logger';
+
+export type LogsSynthtraceEsClientOptions = Omit<SynthtraceEsClientOptions, 'pipeline'>;
+
+export class LogsSynthtraceEsClient extends SynthtraceEsClient<LogDocument> {
+  constructor(options: { client: Client; logger: Logger } & LogsSynthtraceEsClientOptions) {
+    super({
+      ...options,
+      pipeline: logsPipeline(),
+    });
+    this.dataStreams = ['logs-*'];
```

Comment:
Don't know what this exactly does but the data streams we work with are `logs-*-*`, small difference but could be relevant ;-)

Reply:
This is used to cleanup data when we want to cleanup the index after our tests to avoid conflict with other tests, but it's a good catch, i will update it